.github/workflows/build-tarball.yml

```
@@ -2,6 +2,7 @@ name: Build from tarball
 
 on:
   pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
```

Do we need reopened?
```suggestion
    types: [opened, synchronize, ready_for_review]
```

---

I don't know. It's part of the three types enabled by default.

---

I'd keep it for now if it's in the default set. Untested, but maybe it covers the situation:
Open PR ➡️ Accidentally close PR ➡️ Update branch for PR ➡️ Reopen PR

---

+1 to keeping it, it makes sense if we've got `opened` and aren't doing an action only when the PR is created.